src/components/ForecastMainBanner.svelte

```diff
@@ -53,7 +58,7 @@
                   temperature >= 0 ? 'red' : 'blue'
                 }-500 md:hidden m-3`}
               >
-                {temperature}°
+                {temperature}{temperatureSign}
```

Comment:
Good job, I've probably extracted this into separate component <Temperature /> with some props so we dont need to track symbol everywhere and it would be faster to apply changes